### ISSUE TITLE
Unable to open docker after recent update  

### ISSUE THREAD

**Issue by choux130** [docker/for-win#2921]
### Diagnostic id
28DDE6BD-2C9B-48C8-BC86-1A8FA23EDD14/20181120133515

### Expected behavior
Open Docker successfully.

### Actual behavior
When I executed Docker. It gave me error like this, 
```
Unable to create: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Generic failure 
at New-Switch, <No file>: line 121
at <ScriptBlock>, <No file>: line 411
   at Docker.Core.Pipe.NamedPipeClient.Send(String action, Object[] parameters) in C:\workspaces\edge-18.09.x\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeClient.cs:line 36
   at Docker.Actions.DoStart(SynchronizationContext syncCtx, Boolean showWelcomeWindow, Boolean executeAfterStartCleanup) in C:\workspaces\edge-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 92
   at Docker.Actions.<>c__DisplayClass19_0.<Start>b__0() in C:\workspaces\edge-18.09.x\src\github.com\docker\pinata\win\src\Docker.Windows\Actions.cs:line 74
   at Docker.WPF.TaskQueue.<>c__DisplayClass19_0.<.ctor>b__1() in C:\workspaces\edge-18.09.x\src\github.com\docker\pinata\win\src\Docker.WPF\TaskQueue.cs:line 59
```
I have also tried on the Stable version but got the same error. 

This is the result when I run <code> > docker version</code> on Powershell (ran as administrator)
```
Client: Docker Engine - Community
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:47:51 2018
 OS/Arch:           windows/amd64
 Experimental:      false
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.39/version: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.
```

### Information
  - Windows Version: Windows 10 Pro
  - Docker Desktop Version : Version 2.0.0.0-win77 (28777)

### Steps to reproduce the behavior
  1. Update the Docker.
  2. Run it and then get the error.

**Comment by Thierry61** [comment]
Same for me after upgrading to 2.0.0.0-win78 [#2914](https://github.com/docker/for-win/issues/2941)

**Comment by emilgenchev** [comment]
Experiencing same problem with docker Version 2.0.0.0-win81 (29211). 
seems like issue with hyper-V virtual switch.

**Comment by Thierry61** [comment]
In my case the problem is corrected with latest version [Version 2.0.0.3 (31259)]
